src/components/timer/Timer.cpp

```diff
@@ -0,0 +1,28 @@
+#include "components/timer/Timer.h"
+
+using namespace Pinetime::Controllers;
+
+Timer::Timer(void* const timerData, TimerCallbackFunction_t timerCallbackFunction) {
```

Comment:
The `timerData` argument type doesn't match the declaration in Timer.h: here it is `void *const` and in Timer.h it is `void *`.

Reply:
That doesn't matter, it doesn't affect the type of the parameter passed to the function.

Reply:
I know, but it's better to be consistent between declaration and definition.

Reply:
Our clang-tidy configuration disagrees. The argument is that it doesn't affect the function signature. https://clang.llvm.org/extra/clang-tidy/checks/readability/avoid-const-params-in-decls.html. Therefore this is outside the scope of this PR.